README.rst

```diff
@@ -1,132 +1,196 @@
+    $ pip3 install coala-bears
 
-.. Stop ignoring LineLengthBear
+|Stable|
 
-Get *coala* to lint all your languages in your project with one tool and
-config!
+To install the latest development version run:
 
-Demo (Click to View)
---------------------
+.. code-block:: bash
 
-|asciicast|
+    $ pip3 install coala-bears --pre
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
-   :target: https://asciinema.org/a/42968?autoplay=1
-   :width: 100%
+The latest code from the master branch is automatically deployed as the
+development version in PyPI.
+
+|PyPI| |Windows| |Linux|
 
-About
 -----
 
-*coala* is a language independent analysis toolkit. It empowers developers
-to create rules which a project's code should conform to. *coala* takes care
-of showing these issues to users in a friendly manner, is versatile and can be
-used in any environment. Patches to automatically fix code will be managed too.
-*coala* has a set of official bears (plugins) to provide an out-of-the-box
-analysis functionality for many popular languages in addition to some
-generically applicable algorithms.
+=====
+Usage
+=====
 
-To see what coala can do for you and your language, take a look at
-`our capabilities listing <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst>`__.
+To get the list of arguments and their meaning, run:
 
-*coala* is written with a lower case "c".
+.. code-block:: bash
 
-Read more at our `documentation <http://coala.rtfd.org/>`__.
+    $ coala --help
 
-To consult our list of Frequently Asked Questions, simply visit `this link <http://coala.readthedocs.io/en/latest/Users/FAQ.html>`__.
+To run coala over your project, create a ``.coafile``, the project-specific
+configuration file that will store all your settings for coala. A sample
+``.coafile`` will look something like this:
 
-Why use coala?
---------------
+.. code-block:: bash
 
-- *coala* provides **built-in checking routines** (named bears in *coala*).
-- **Serves your requirement**: You can easily write your own checks (using
-  bears).
-- *coala* provides **user-friendly interfaces** like json, formatted and
-  interactive output in the CLI and *plugins for various editors* are
-  available as well.
-- **Optimal performance**: *coala* manages parallelizing the checking-routines
-  without you having to worry.
-- **Unified interface**: One tool for all programming languages.
+    [Spacing]
+    files = src/**/*.py
+    bears = SpaceConsistencyBear
+    use_spaces = True
 
-If you want to learn more about *coala*, its functionality and its usage,
-please take a look at our
-`tutorial <http://coala.rtfd.org/en/latest/Users/Tutorials/Tutorial.html>`__.
+* The `files` key tells coala which files to lint - here, we're linting all
+  python files inside the `src` directory.
+* The `bears` key specifies which bears (plugins) to use. We support a huge
+  number of languages and you can find the whole list
+  `here <github.com/coala-analyzer/bear-docs>`__.
```

Comment:
If your language is not there, don't worry, we got some bears that work for any language (link "bears that work for any language" to the bear docs section with the general bears) or you can file an issue (https://github.com/coala-analyzer/coala-bears/issues/new) and we will do that for you!